frontend/src/store/reducers/UserReducer.js

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
For now, I think this file shouldn't be included in this PR.

Reply:
Thank you!